akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1460,12 +1460,41 @@ trait FlowOps[+Out, +Mat] {
    *
    * '''Cancels when''' downstream cancels
    *
-   * @param seed Provides the first state for extrapolation using the first unconsumed element
    * @param extrapolate Takes the current extrapolation state to produce an output element and the next extrapolation
    *                    state.
    */
   def expand[U](extrapolate: Out ⇒ Iterator[U]): Repr[U] = via(new Expand(extrapolate))
 
+  /**
+   * Allows a faster downstream to progress of a slower upstream.
+   *
+   * This is achieved by introducing "extrapolated" elements - based on those from upstream - whenever downstream
+   * signals demand.
```

Comment:
It should be even more clear that if there is enough elements flowing from upstream the extrapolation is not invoked, and "if you want to always have values pass through the extrapolate function, see [[link to old method]]"

Reply:
+1, see later comment.